catalyst/dl/core/runner.py

```diff
@@ -107,7 +107,7 @@ def _run_batch(self, batch):
 
         self._run_event("batch_start")
         self.state.timer.start("_timers/model_time")
-        self.state.output = self.predict_batch(batch)
+        self.state.output = self.forward(batch)
```

Comment:
you still need to pass batch to device, so `predict_batch` required

Reply:
Nope, batch moved to device few lines above in the same function

Reply:
okay, i see
than, could you please move `forward` method on top of `predict_batch`?

like
```
def forward(...)
   ...

def predict_batch(...)
    ...
    self.forward()

def _run_batch(...):
    ...
    self.forward()
```